### DISCUSSION TITLE
Jenkins Errors for Provision Jobs - Provision/Core/MongodbCluster

### DISCUSSION THREAD

**Discussion by bsimmonayyakutti** [project-sunbird/sunbird-devops#3498] [General]
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/SIMMON~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/SIMMON~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

TASK   [mongodb-cluster : Install MongoDB package]   *******************************     fatal: [10.128.0.23]: FAILED! => {         "changed": false     }          MSG:          Failed to update apt cache: W:Updating from such a repository can't be done   securely, and is therefore disabled by default., W:See apt-secure(8) manpage   for repository creation and user configuration details., W:Target Packages   (multiverse/binary-amd64/Packages) is configured multiple times in   /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and   /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, W:Target Packages   (multiverse/binary-all/Packages) is configured multiple times in   /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and   /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, W:Target   Translations (multiverse/i18n/Translation-en) is configured multiple times in   /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and   /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, W:Target CNF   (multiverse/cnf/Commands-amd64) is configured multiple times in   /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and   /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, W:Target CNF   (multiverse/cnf/Commands-all) is configured multiple times in   /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and   /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, E:The repository   'http://www.apache.org/dist/cassandra/debian 39x Release' does not have a   Release file.               PLAY RECAP   *********************************************************************     10.128.0.23                :   ok=6    changed=0    unreachable=0    failed=1    skipped=4    rescued=0    ignored=0
--

</body>

</html>

**Comment by dikshauat** [comment]
Hi, I am also getting the same error. Please let me know if you find any resolve for it. 

Regards
